Birthday-2024-Project/Scripts/UI/Puzzle/States/PuzzleUiWinState.gd

```diff
@@ -31,6 +32,7 @@ func enter_state():
 
 func exit_state():
 	var screen = _ui_manager.main_screen
+	screen.skip_button.set_theme(load("res://Art/UI/Themes/Next_Level_Button_theme.tres"))
```

Comment:
Should probably switch it back to Skip Button theme on exit.
Or, set some flag so if the player decides to redo the level, it doesn't prompt a skip confirmation.